sitemap/s3saver.go

```diff
@@ -38,5 +40,6 @@ func (s *S3Saver) SaveFile(lang config.Language, body io.Reader) error {
 	if err != nil {
 		return fmt.Errorf("failed to upload file to s3: %w", err)
 	}
+	log.Info(context.Background(), fmt.Sprintf("saved key [%s], bucket [%s]", k, s.bucket))
```

Comment:
I don't think that the key should be logged due to security?

Reply:
the key is just the language english/welsh :)